### ISSUE TITLE
MIM in pre-train

### ISSUE THREAD

**Issue by dt-yuhui** [birkhoffkiki/GPFM#16]
Thanks for your great work on this project.

I'm a bit confused about the MIM in the pre-training phase. Is it a reconstruction loss, or does it involve predicting masked tokens similar to iBOT?

I would appreciate it if you could show me where in the code this is handled.

**Comment by birkhoffkiki** [comment]
Yes, similar to iBOT, you can find it at:
https://github.com/birkhoffkiki/GPFM/blob/3a750fb2ae5ab2004c0a0f44ef63920a3cc890c6/pretrain/dinov2/train/ssl_meta_arch.py#L138

**Comment by dt-yuhui** [comment]
thanks for your quick reply
